lib/benchee.ex

```diff
@@ -55,6 +55,14 @@ for {module, moduledoc} <- [{Benchee, elixir_doc}, {:benchee, erlang_doc}] do
 
     defp add_benchmarking_jobs(suite, jobs) do
       Enum.reduce(jobs, suite, fn {key, function}, suite_acc ->
+        if is_function(function) && :erlang.fun_info(function, :module) == {:module, :erl_eval} do
```

Comment:
As you said you didn't deal with the tuple version here. I think we need to deal with it though, although it's not often used it should still work :)

Iirc it is `{function, kwlist}` so a pattern match to extract the function we want to check should be rather easy.